pubpub/platform
# Persist action instance config

Adds a rough version of action instance config saving.
- Adds a new JSONB column for config to the action_instances table
- Adds a submit button to the action instance config form which saves the config in the new column

## Issue(s) Resolved
#296 
## Test Plan
- Regenerate your prisma client
- Open http://localhost:3000/c/unjournal/stages/manage and click the gear icon in a stage
- Select the actions tab in the stage panel
- Add any action
- Click the newly added action and its form should open
- Fill it out and submit, then the action's config should be persisted to the db

### Notes/Context/Gotchas

I'll open a followup PR with some additional (mostly UX) improvements such as:
- Make the tabs panel scrollable
- Render the config form when actions are being added initially if they have required config
- Make the submit button match our other button styles and animations
- Stable sort order for actions in the panel (currently if you edit an action's config, it might move after you save it)

I'd also like to add names to the action instances so they can be distinguished without looking at their config
